src/util-misc.c

```diff
@@ -126,11 +118,11 @@ static int ParseSizeString(const char *size, double *res)
             goto end;
         }
 
-        if (strcasecmp(str2, "kb") == 0) {
+        if (strcasecmp(str2, "kb") == 0 || strcasecmp(str2, "KiB") == 0) {
             *res *= 1024;
-        } else if (strcasecmp(str2, "mb") == 0) {
+        } else if (strcasecmp(str2, "mb") == 0 || strcasecmp(str2, "MiB") == 0) {
             *res *= 1024 * 1024;
-        } else if (strcasecmp(str2, "gb") == 0) {
+        } else if (strcasecmp(str2, "gb") == 0 || strcasecmp(str2, "GiB") == 0) {
```

Comment:
on further thought, I think the new options should be case sensitive as `MiB` != `Mib`. Uppercase means bytes, lowercase bits. Needs update to test and error messages then too.